src/FirstAssignment/CharactersEqualOccurrences.java

```diff
@@ -3,5 +3,33 @@
 
 public class CharactersEqualOccurrences {
 
+	public static void main(String[] args) {
+		boolean b1;
+		b1 = areOccurrencesEqual("abacbc");
+		System.out.println("String abacbc is : "+b1);
+		b1 = areOccurrencesEqual("aaabb");
+		System.out.println("String aaabb is : "+b1);
+	}
+
+	public static boolean areOccurrencesEqual(String s) { 
+
+		int[] counts = new int[26];
```

Comment:
format the code

Reply:
Added formatting in the code and Pushed code to same branch